### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by pedrocamargo**
Minor updates to bring the recommended version to 3.8 an a few more things left over from old documentation

### CODE REVIEW

File: aequilibrae/project/network/node.py [hunk 1]
```diff
@@ -91,8 +91,6 @@ def renumber(self, new_id: int):
 
         curr.execute("BEGIN;")
         curr.execute("Update Nodes set node_id=? where node_id=?", [new_id, self.node_id])
-        curr.execute("Update Links set a_node=? where a_node=?", [new_id, self.node_id])
```
Comment (jamiecook) [inline aequilibrae/project/network/node.py:94]: this doesn't look like documentation change?

File: docs/source/gettingstarted.rst [hunk 2]
```diff
@@ -115,13 +119,13 @@ things to keep an eye on are:
 
 * Number of zones on your model (size of the matrices you are dealing with)
 
-* Number of matrices (classes you are dealing with)
+* Number of matrices (vehicles classes (and user classes) you are dealing with)
```
Comment (jamiecook) [inline docs/source/gettingstarted.rst:122]: double parens

**Review by jamiecook — APPROVED** [review]

File: aequilibrae/project/network/node.py (hunk 1 above)
Comment (pedrocamargo) [reply aequilibrae/project/network/node.py:94]: Not indeed.  But realized that this was covered by a trigger

File: docs/source/gettingstarted.rst (hunk 2 above)
Comment (pedrocamargo) [reply docs/source/gettingstarted.rst:122]: Yeah...  A bit lazy...

**Pull request merged**